src/sentry/static/sentry/app/stores/guideStore.tsx

```diff
@@ -131,12 +131,13 @@ const guideStoreConfig: Reflux.StoreDefinition & GuideStoreInterface = {
   },
 
   onCloseGuide() {
-    const {currentGuide} = this.state;
-    this.state.guides.map(guide => {
```

Comment:
Not sure why were using map for something that's really a filter + side effect lol